src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -36,28 +36,35 @@
 
 .bcs-VersionsItem-details {
     flex: 1 1 100%;
+    margin-right: 35px; // Avoid flowing under action toggle button
```

Comment:
I believe this will cause the "info" content to also be truncated, which isn't desirable.